crashDataAnalysisTools/plot_and_save.py

```diff
@@ -0,0 +1,67 @@
+import os
+import pandas as pd
+from matplotlib.backends.backend_pdf import PdfPages
+
+
+def PlotXVsY(df, colname_x, colname_y):
```

Comment:
For this and other plotting functions, it may be useful to implement dynamic plots where ranges of values to be plotted etc. are controlled via ipython widgets.